File: photoMe_api/Models/PhotoShoot.cs
using System;

namespace photoMe_api.Models
{
    public class PhotoShoot : BaseModel
    {
        public Guid? PhotographerId { get; set; }
        public User Photographer { get; set; }
        public Guid? ModelId { get; set; }
        public User Model { get; set; }
        public string Location { get; set; }
        public int Price { get; set; }
        public string ShootTime { get; set; }
        public DateTime ShootDate { get; set; }
        public string MeetingPlaceDetail { get; set; }
        public string MeetingPlace { get; set; }
        public string AdditionalInfo { get; set; }
        public string PaymentMethod { get; set; }
        public AdditionalService AdditionalService { get; set; }
        public Guid? PackageId { get; set; }
        public virtual Package Package { get; set; }
        public string Status { get; set; }
    }
}

File: photoMe-web/src/app/shared/models/PhotoShoot.ts
import { Package } from './Package';

export class PhotoShoot {
  package: Package;
  location: string;
  price: number;
  shootTime: string;
  shootDate: Date;
  meetingPlace: string;
  meetingPlaceDetail: string;
  additionalInfo: string;
  modelId: string;
  photographerId: string;
  additionalService: string;
  paymentMethod: string;
  modelName: string;
  photographerName: string;
  status: string;

  constructor(
    addtionalInfo: string,
    addtionalService: string,
    meetingPlace: string,
    meetingPlaceDetail: string,
    shootDate: Date,
    shootTime: string
  ) {
    this.additionalInfo = addtionalInfo;
    this.additionalService = addtionalService;
    this.meetingPlace = meetingPlace;
    this.meetingPlaceDetail = meetingPlaceDetail;
    this.shootTime = shootTime;
    this.shootDate = shootDate;
  }
}
